.github/workflows/diff_report.yml

```diff
@@ -104,6 +104,17 @@ jobs:
         ref: master
         path: checkstyle
         fetch-depth: 0
+
+     # set main checkstyle repo as an upstream
+     # Diff report will be generated taking upstream's master branch as a base branch
+     - name: set upstream
+       run: |
+         cd checkstyle
+         bash
+         MAIN_REPO_GIT_URL="https://github.com/checkstyle/checkstyle.git"
+         git remote add original_checkstyle $MAIN_REPO_GIT_URL\
+           && git fetch original_checkstyle;
```

Comment:
please do `git fetch original_checkstyle` as separate command, not need for long lines,
We use model - fail fast. If something is failing to execute, all commands below unlikely to execute too.

Reply:
OK, Done